meta/classes/build_yocto.bbclass

```diff
@@ -22,11 +22,14 @@ EXPANDED_XT_SSTATE_CACHE_MIRROR_DIR = "${@d.getVar('XT_SSTATE_CACHE_MIRROR_DIR')
 EXPANDED_XT_ALLOW_SSTATE_CACHE_MIRROR_USE = "${@d.getVar('XT_ALLOW_SSTATE_CACHE_MIRROR_USE') or ''}"
 EXPANDED_XT_POPULATE_SDK = "${@d.getVar('XT_POPULATE_SDK') or ''}"
 
+REAL_XT_BB_CONFIG_CMD = "${@d.getVar('XT_BB_CONFIG_CMD') or 'source poky/oe-init-build-env'}"
+REAL_XT_BB_RUN_CMD = "${@d.getVar('XT_BB_RUN_CMD') or 'source poky/oe-init-build-env'}"
```

Comment:
Could you please clafiry why you introduced two commands identical. Both of them just setup an environment for bitbake calling.

Reply:
yes, first one (REAL_XT_BB_CONFIG_CMD) is smart enough in case of AGL (e.g. it can be run multiple times and will not overwrite local.conf and bblayers.conf files), but to be on the safe side I introduce one more command.
Use case:
1. You run some_distro_setup.sh and then you can run bitbake to bake bits
2. In our case we need to run bitbake-layers in other shell (other build task will clear environment, so we need to source it again).
3. If setup script from 1) cannot be run multiple times (it overwrites config each time) then we may loose all the changes to local.conf/bblayers.conf we did.
4. For that reason we may want to provide "source env" command instead